curp-testutils/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "curp-testutils"
```

Comment:
I recommend `curp-test-utils` since utils is an initialism.

Reply:
done